theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-contribution.tsx

```diff
@@ -82,6 +84,18 @@ export class TraceViewerToolbarContribution implements TabBarToolbarContribution
                 signalManager().fireResetZoomSignal();
             }
         });
+        registry.registerCommand(
+            TraceViewerToolbarCommands.OPEN_TRACE, {
+            isVisible: (w: Widget) => {
+                if (w instanceof TraceExplorerOpenedTracesWidget) {
+                    return true;
+                }
+                return false;
+            },
+            execute: () => {
+                registry.executeCommand(OpenTraceCommand.id);
```

Comment:
`await` here to, right? If yes, you will need to change line 95 to add async modifier (`execute: async`)